app/assets/javascripts/creator.js

```diff
@@ -1,5 +1,3 @@
 Blacklight.onLoad( function() {
-	var authoritySelect = require('authority_select');
-	var as = new authoritySelect.AuthoritySelect({ selectBox : "select#work_creator", inputField : "input.work_creator" });
-	as.initialize();
+	Hyrax.authoritySelect({ selectBox : "select#work_creator", inputField : "input.work_creator" });
```

Comment:
`#work_creator` is not a unique identifier on the page.  because you call `f.input key` twice here: https://github.com/curationexperts/ldc-qa-demo/blob/d00d6cba8539711efe1b0dea073483c61850aa8f/app/views/records/edit_fields/_creator.html.erb#L13   The HTML5 spec says that the ID *MUST* be unique https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute So, for starters, how about `select.work_creator`?